### PR TITLE
DDP-6767: Fix shifted footer border for screen 1024px [PANCAN]

### PR DESCRIPTION
**Pull request by alextanlan**
**_Before_**

**_After_**

### CODE REVIEW

File: ddp-workspace/projects/ddp-pancan/src/app/components/footer/footer.component.scss [hunk 1]
```diff
@@ -61,7 +61,7 @@
     min-width: fit-content;
 }
 
-@media screen and (min-width: $desktop-breakpoint) {
+@media screen and (min-width: $desktop-breakpoint + 1) {
```
Comment (aglushko-broad) [inline ddp-workspace/projects/ddp-pancan/src/app/components/footer/footer.component.scss:64]: what do you think if we replace mobile first(min-width) with desktop first (max-width) here?

Comment (alextanlan) [reply ddp-workspace/projects/ddp-pancan/src/app/components/footer/footer.component.scss:64]: Not sure if I get the point.
we already have 
```  
@media screen and (max-width: $desktop-breakpoint) {  // line 79
   ...change flex direction inside of some footer elements
}
 ```


The layout issue appears for the exact value `1024px`.

Comment (aglushko-broad) [reply ddp-workspace/projects/ddp-pancan/src/app/components/footer/footer.component.scss:64]: I mean this
https://github.com/broadinstitute/ddp-angular/pull/816/files
what do you think?

Comment (alextanlan) [reply ddp-workspace/projects/ddp-pancan/src/app/components/footer/footer.component.scss:64]: Deal, merged.
Will close this PR as redundant.

**Pull request closed without merging**